site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -4668,6 +4669,35 @@ public void doSite_search_clear(RunData data, Context context) {
 
 	} // doSite_search_clear
 
+	public void doUser_search(RunData data, Context context) {
+		SessionState state = ((JetspeedRunData) data)
+				.getPortletSessionState(((JetspeedRunData) data).getJs_peid());
+
+		// read the search form field into the state object
+		String search = StringUtils.trimToNull(data.getParameters().getString(
```

Comment:
I think this needs escaping for XSS filtering, otherwise someone can inject javascript into a WebLearn URL.